example/src/main.rs

```diff
@@ -0,0 +1,33 @@
+extern crate civet;
+extern crate conduit;
+extern crate conduit_router;
+
+use std::collections::HashMap;
+use std::io::{self, Cursor};
+use std::sync::mpsc::channel;
+
+use civet::{Config, response, Server};
+use conduit::{Request, Response};
+use conduit_router::{RouteBuilder, params};
+
+
+fn name(req: &mut Request) -> io::Result<Response> {
+    let name = params(req).find("name").unwrap();
+    let bytes = format!("Hello {}!", name).into_bytes();
+
+    Ok(response(200, HashMap::new(), Cursor::new(bytes)))
+}
+
+
+fn main() {
+    let mut router = RouteBuilder::new();
+
+    router.get("/:name", name);
+
+    let router = router;
```

Comment:
Ah I think in the past I've used this when there's a large amount of code between the "freeze point" and the consumption point to ensure I don't change the router again, but in this case that distance is so short it may not be worth it :)